### DISCUSSION TITLE
Designer onboarding

### DISCUSSION THREAD

**Discussion by almirsarajcic** [3Stones-io/omedis#372] [General]
@Happiartista, as promised, here is a screenrecording and some screenshots.

@kagure-nyakio, please meet with Oghogho tomorrow and walk her through the app.
Show her different pages with different states. For some pages, Today view especially, create some data in the DB manually to show what it looks like when there's some data.
Then you can use that to update demo seeds so we can all have that data available on our machines.

https://github.com/user-attachments/assets/362a8242-6639-4b39-b624-5e5783eb508c

<img width="881" alt="Screenshot 2024-11-21 at 18 39 04" src="https://github.com/user-attachments/assets/2a8269db-625a-4452-b84c-4ddf0e7ac73d">

<img width="1728" alt="Screenshot 2024-11-21 at 18 38 15" src="https://github.com/user-attachments/assets/75805b7e-ee49-4396-8888-83d18b181b8e">

<img width="571" alt="Screenshot 2024-11-21 at 18 38 45" src="https://github.com/user-attachments/assets/d9ebf302-12f8-4a26-bacc-8d5a935d3057">

**Comment by fradarr** [comment]
As to complement our discussion from the last week in direction of "minimalistic" cleen design here two more examples,
1) GitHub Menu 


https://github.com/user-attachments/assets/9e41c1ab-e215-4a0c-92e4-49686c17f8df



2) GitHub Projects Open Close Toggle 

https://github.com/user-attachments/assets/6e88b5d8-78b6-4022-907e-4940f067e009

3) DuckDuckGo

https://github.com/user-attachments/assets/c07c98b4-5ed7-42bb-97c0-62b13bc34e0c
